### PR TITLE
feat: add react-widget to storybook

### PR DESCRIPTION
**Pull request by AdriSolid**
For: https://app.clubhouse.io/cartoteam/story/141940/storybook-detailed-revision

### CODE REVIEW

File: packages/react-ui/storybook/.storybook/preview.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ export const parameters = {
         ['Palette', 'Typography'],
         'Common',
         'Widgets',
-        ['WrapperWidgetUI', 'FormulaWidgetUI', 'CategoryWidgetUI']
+        ['FormulaWidget', 'CategoryWidget', 'HistogramWidget', 'PieWidget'],
+        'WidgetsUI',
+        ['CategoryWidgetUI']
```
Comment (VictorVelarde) [inline packages/react-ui/storybook/.storybook/preview.js:53]: @AdriSolid aren't we losing here the other UI widgets?

File: packages/react-ui/storybook/stories/Introduction.stories.mdx [hunk 2]
```diff
@@ -2,36 +2,43 @@ import { Meta, Description } from '@storybook/addon-docs/blocks';
 
 <Meta title="Introduction" />
 
-# CARTO for React - UI Components
+# CARTO for React - Widgets and UI Components
 This storybook includes the UI Components for CARTO for React (@carto/react-ui).
```
Comment (VictorVelarde) [inline packages/react-ui/storybook/stories/Introduction.stories.mdx:6]: `This storybook includes the UI Components for CARTO for React (@carto/react-ui) and the powerful Widgets (@carto/react-widgets).`  Adding something like that?

File: packages/react-ui/storybook/.storybook/preview.js (hunk 1 above)
Comment (AdriSolid) [reply packages/react-ui/storybook/.storybook/preview.js:53]: Nop, that's actually the order, let me add same order as `Widget`s :+1:

File: packages/react-ui/storybook/stories/Introduction.stories.mdx (hunk 2 above)
Comment (AdriSolid) [reply packages/react-ui/storybook/stories/Introduction.stories.mdx:6]: LGTM! Little change..
`This storybook includes the UI Components (@carto/react-ui) and the powerful Widgets (@carto/react-widgets) for CARTO for React`
What you think?

File: packages/react-widgets/src/widgets/CategoryWidget.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ import useWidgetLoadingState from './useWidgetLoadingState';
  * @param  {string} [props.operationColumn] - Name of the data source's column to operate with. If not defined it will default to the one defined in `column`.
  * @param  {string} props.operation - Operation to apply to the operationColumn. Must be one of those defined in `AggregationTypes` object.
  * @param  {formatterCallback} [props.formatter] - Function to format each value returned.
+ * @param  {Object} [props.labels] - Overwrite category labels
```
Comment (VictorVelarde) [inline packages/react-widgets/src/widgets/CategoryWidget.js:24]: Good catch

File: packages/react-widgets/src/widgets/PieWidget.js [hunk 4]
```diff
@@ -153,7 +153,8 @@ PieWidget.propTypes = {
   formatter: PropTypes.func,
   tooltipFormatter: PropTypes.func,
   viewportFilter: PropTypes.bool,
-  onError: PropTypes.func
+  onError: PropTypes.func,
+  wrapperProps: PropTypes.object
```
Comment (VictorVelarde) [inline packages/react-widgets/src/widgets/PieWidget.js:157]: we're we missing this? nice catch.

Comment (AdriSolid) [reply packages/react-widgets/src/widgets/PieWidget.js:157]: Indeed :S

**Review by VictorVelarde — APPROVED** [review]
Add a small note to changelog about adding widgets to storybook and 🚀

**Pull request merged**